src/main/java/applicationPackage/Application.java

```diff
@@ -0,0 +1,44 @@
+package applicationPackage;
+
+import java.beans.XMLEncoder;
+import java.io.BufferedOutputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.TreeSet;
+
+public class Application {
+    private final static String DECODED_FILE_NAME = "decod.txt";
+
+    public static void main(String[] args) {
```

Comment:
better move main method to the separate app class

Reply:
What should I do with the main method class. Should I remove all the methods from this class beside mainMethod? Even a technical method to create a list to decode?